tests/test_kdd.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+export NODENAME="kube-master"
+
+echo "Waiting for API server to come online"
+until /bin/kubectl version; do
+  sleep 1
+  done
+
+/bin/kubectl apply -f /tests/tprs.yaml
+/bin/kubectl apply -f /tests/nodes.yaml
+
+echo "Waiting for TPRs to apply"
+until /bin/kubectl apply -f /tests/tpr_resources.yaml; do
```

Comment:
The TPRs take some time to be available for use once they are created, so we just keep trying until it takes them, which is a few seconds.